### PR TITLE
Introduce useExamples in v1 version

### PR DESCRIPTION
**Pull request by Remco75**
Same functionality as in #39, see discussion in #33, but finally I got it in version 1.x

### CODE REVIEW

**Comment by subeeshcbabu** [comment]
Nice 👏 . Thanks for the PR

**Comment by Remco75** [comment]
Thnx for merging! Could you release this version to NPM by any chance? Then I can fix my package to use the npm version once more :-)

Thnx!

**Comment by subeeshcbabu-zz** [comment]
Published `swagmock@1.1.0-alpha.1`. Can you plz check and confirm if it works.

File: lib/generators/index.js [hunk 1]
```diff
@@ -3,26 +3,31 @@ const Chance = require('chance').Chance();
...
     if (schema) {
         let type = schema.type || findType(schema);
+        let example = schema.examples || schema.example;
         /**
          * Get the mock generator from the `type` of the schema
          */
-        const generator = Generators[type];
-        if (generator) {
-            mock = generator.call(null, schema);
+        if (example && useExample) {
```
Comment (sammarks) [inline lib/generators/index.js:14]: We should probably use `if (typeof example !== 'undefined' && useExample)` here instead. I have some boolean property examples as `false` and that's causing this to generate random values (sometimes true!) for these instead.

**Pull request merged**